pkg/rancher-desktop/main/tray.ts

```diff
@@ -145,14 +149,14 @@ export class Tray {
         }
       }
 
-      abortController.abort();
+      this.abortController.abort();
```

Comment:
Don't we need to reset the abort controller at some point (so that it can be used again)?